### DISCUSSION TITLE
Using OneDrive as remote drive

### DISCUSSION THREAD

**Discussion by gustavopi** [abraunegg/onedrive#1411] [Q&A] [answered]
My expectation of onedrive was the use similar to my Windows and macOS: the file list is there but the files are in the cloud, only files used recently are locally available. I have 1TB in my plan and my volumes are small, Windows got only 100GB. So I am using it more like a remote disk than cloud working.
So my suggestion is to make it available, a monitor option to get only file list and download only what is to be used.
I am still learning how onedrive works so maybe it is possible that something like this is already available - please tell me.
Finally, of course, congratulations for this awesome!

**Comment by abraunegg** [comment, marked as answer]
@gustavopi 
This client is a 'sync' client - it syncs your data.

The feature you are wanting to replicate is the 'on-demand' feature of of OneDrive. See #757 for details or use https://github.com/jstaf/onedriver
